kano/utils/audio.py

```diff
@@ -62,26 +69,15 @@ def get_volume():
     from kano.logging import logger
 
     percent = 100
-    millibel = 400
 
-    cmd = "amixer | grep %"
-    o, _, _ = run_cmd(cmd)
-    o = o.strip().split(' ')
-
-    try:
-        millibel = int(o[2])
-    except Exception:
-        msg = 'asmixer format bad for millibel, o: {}'.format(o)
-        logger.error(msg)
-        pass
+    cmd = "amixer | head -n 6 | grep -Po '(\d{1,3})(?=%)'"
```

Comment:
I would say that for the time being, this is sufficient. The `Master` channel seems to always be reported first. We can have tests that look out for this if it ever fails. And if we ever add other channels or rename `Master`, chances are that there will be more changes than this.

If you'd still like a more complicated approach, I could have a look.